ball.py

```diff
@@ -103,6 +103,8 @@ def __init__(self, sprites, filename):
         self.ball.set_layer(3)
         self.ball.set_label_attributes(24, vert_align='top')
 
+        self.grabbed = False
```

Comment:
It just makes logical sense to me that the ball is the thing being grabbed so it could be it's property, if ball ever needs to access it in the future for, say a texture change when being grabbed, even then it could be used. If you think it would be better suited as an attribute of the activity class. please let me know, I will shift it to the activity class.

Reply:
Nothing in the class uses it.  That's the reason to change.  Having moved it to the activity class, it could be `drag`, `grab` or `mouse_grab` instead of `grabbed`.